src/sentry/api/serializers/models/rule.py

```diff
@@ -195,4 +197,10 @@ def serialize(self, obj, attrs, user, **kwargs):
         else:
             d["snooze"] = False
 
+        org = Organization.objects.get(id=obj.project.organization_id)
+        neglected_rule = NeglectedRule.objects.filter(rule=obj, organization=org, opted_out=False)
```

Comment:
it might make sense to use .first since i think .exists makes its own query?
```suggestion
        neglected_rule = NeglectedRule.objects.filter(rule=obj, organization=org, opted_out=False).first()
```